### PR TITLE
typo fix

### PR DESCRIPTION
**Pull request by jonyhy96**
typo fix
```release-note
None
```

### CODE REVIEW

File: stable/quassel/README.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ $ helm install stable/quassel --name my-release -f values.yaml
 ## Persistence
 
 The [quassel-core](https://hub.docker.com/r/linuxserver/quassel-core) image
-stores it's configuration data, and if using SQLite, it's SQLite datbase at the
+stores it's configuration data, and if using SQLite, it's SQLite datebase at the
```
Comment (timstoop) [inline stable/quassel/README.md:97]: You have a typo in your typo fix :) It should be `database`, ofcourse.

**Review by timstoop — CHANGES_REQUESTED** [review]

File: stable/quassel/README.md (hunk 1 above)
Comment (jonyhy96) [reply stable/quassel/README.md:97]: Thanks for your correct!

**Comment by jonyhy96** [comment]
Commit is a typo 
Close it

**Pull request closed without merging**